Umbraco-Cloud/Deployment/Umbraco-Forms-on-Cloud/index.md

```diff
@@ -4,62 +4,63 @@ versionFrom: 7.0.0
 
 Read more about the product in the [Umbraco Forms section](../../../Add-ons/UmbracoForms).
 
-## How are Forms handled on Cloud?
+## How are forms handled on Cloud?
 
-Umbraco Forms are handled as content and media. This means that you can transfer your Forms between environments using the same workflow you use for content and media.
+Forms are handled like content and media. This means that you can transfer your forms between environments, using the same workflow you use for content and media.
 
 Entries submitted are not transferred to the next environment, as they are *environment specific*. If you need to move entries from one environment to another, you need to run an export/import script on the databases.
 
 ## Recommended workflow
 
-You can work with Forms on the environment of your choice.
+You can work with forms on the environment of your choice.
 
-When you need to test or use your forms on another environment you can
+When you need to test or use your forms on another environment you can:
 
 * Transfer the forms to the next environment using **Queue for transfer** or
 * **Restore** the forms on an environment lower in the workflow
 
-For more information on how to handle content transfer / restores on Umbraco Cloud, checkout the following articles:
+For more information on how to handle content transfer/restores on Umbraco Cloud, checkout the following articles:
 
-* [Transfer content, media and Forms](../Content-Transfer)
+* [Transfer content, media and forms](../Content-Transfer)
 * [Restoring content](../Restoring-content)
 
 ### Did you create your Cloud project before June 18th 2019?
 
-Then your Forms data are handled as meta data. When you create a Form, a UDA file will be generated. This UDA file will contain all the metadata from your form, and be very similar to the `JSON` file that is also generated when you create a new form - this file can be found in `~/App_Data/UmbracoForms/Data/forms`.
+Then your forms data is handled as metadata. When you create a form, an `UDA` file will be generated. This `UDA` file contains all the metadata from the form and is very similar to the `JSON` file that is also generated when you create a new form (found in `App_Data\UmbracoForms\Data\forms`).
 
-This means that your Forms will be deployed along with the rest of your metadata and structure files, e.g. Document Types, Templates and Stylesheets.
+This means that your forms will be deployed along with the rest of your metadata and structure files, e.g. document types, templates and stylesheets.
 
-We strongly recommend that you work with the Forms on your local or Development environment, following the [left-to-right deployment model](../../Deployment).
+We strongly recommend that you work with the forms on your local or development environment, following the [left-to-right deployment model](../../Deployment).
```

Comment:
```suggestion
We strongly recommend that you work with the forms on your local or Development environment, following the [left-to-right deployment model](../../Deployment).
```

Reply:
As this is the name of the exact environment (and you can only have a single 'development' environment), capitalizing seems fine.

Reply:
Yeah, in this context "Development" is the name of a particular environment.
Sorry for not adding an explanation to the suggestion here!